acquia_lift_profiles/js/acquia_lift_profiles.js

```diff
@@ -232,12 +198,20 @@ var _tcwq = _tcwq || [];
             'trackingId': trackingId
           }, settings.acquia_lift_profiles.pageContext, pageFieldValues);
           _tcwq.push(['onLoad', segmentCallback]);
-          _tcaq.push(['captureView', 'Content View', pageInfo]);
 
-          if(settings.acquia_lift_profiles.hasOwnProperty('identity')) {
-            pushCaptureIdentity(settings.acquia_lift_profiles.identity, settings.acquia_lift_profiles.identityType);
+          // Capture view.
+          var captureViewEvent = ['captureView', 'Content View', pageInfo];
+
+          // Capture identity in addition, if applicable.
+          if(!identityCaptured && settings.acquia_lift_profiles.hasOwnProperty('identity')) {
```

Comment:
Do you also have to check settings.acquia_lift_profiles.captureIdentity like you do below on line 258.  Not quite sure what the difference is between the "identity" and "captureIdentity" settings and why one is checked here and the other there.  Probably me not remembering the module well enough - but maybe a comment to explain?